src/Talis/Persona/Client/Tokens.php

```diff
@@ -3,9 +3,14 @@
 
 use \Firebase\JWT\JWT;
 
-class ScopesNotDefinedException extends \Exception
-{
-}
+use Talis\Persona\Client\ScopesNotDefinedException;
```

Comment:
I realise there's all sort of discussion about the right way to do this, but given that you made an absolute import above starting with a `\` (`\Firebase`), I'd suggest you make this consistent and `use \Talis\...`.